components/web_discovery/browser/value_transform_unittest.cc

```diff
@@ -61,6 +108,37 @@ TEST(WebDiscoveryValueTransformTest, FilterExactTransform) {
   EXPECT_FALSE(invalid_transform);
 }
 
+TEST(WebDiscoveryValueTransformTest, RemoveParamsTransform) {
+  // Test removeParams
+  auto transform = CreateTransform(
+      R"(["removeParams", ["utm_source", "utm_medium", "fbclid"]])");
+  ASSERT_TRUE(transform);
+
+  // Test URL with params to remove
```

Comment:
Another one to show that we're not urldecoding the parameters:
```
https://example.com?utm_source=google&utm_medium=cpc&redirect=https%3A%2F%2Fexample.net%3Futm_source%3Dbing%26utm_medium%3Dcpm
```
The expected output is https://example.com?redirect=https%3A%2F%2Fexample.net%3Futm_source%3Dbing%26utm_medium%3Dcpm